### PR TITLE
[CAT] Challenge Platform Limitations

### PR DESCRIPTION
**Pull request by patriciasantaana**
PCX-8942

### CODE REVIEW

File: content/firewall/cf-firewall-rules/cloudflare-challenges.md [hunk 1]
```diff
@@ -153,4 +153,13 @@ You can add tracking scripts to challenge pages to capture the correct referer h
 
 ### Cross-origin resource sharing (CORS) preflight requests
 
-Cross-origin resource sharing (CORS) preflight requests, or `OPTIONS`, excludes user credentials that include cookies. As a result, the `cf_clearance` cookie will not be sent with the request, causing it to fail to bypass a challenge page (non-interactive, managed, or interactive challenge).
\ No newline at end of file
+Cross-origin resource sharing (CORS) preflight requests, or `OPTIONS`, excludes user credentials that include cookies. As a result, the `cf_clearance` cookie will not be sent with the request, causing it to fail to bypass a challenge page (non-interactive, managed, or interactive challenge).
```
Comment (ranbel) [inline content/firewall/cf-firewall-rules/cloudflare-challenges.md]:
```suggestion
Cross-origin resource sharing (CORS) preflight requests, or `OPTIONS`, exclude user credentials that include cookies. As a result, the `cf_clearance` cookie will not be sent with the request, causing it to fail to bypass a challenge page (non-interactive, managed, or interactive challenge).
```

File: content/firewall/cf-firewall-rules/cloudflare-challenges.md [hunk 2]
```diff
@@ -153,4 +153,13 @@ You can add tracking scripts to challenge pages to capture the correct referer h
...
+
+---
+
+## Limitations
+
+Cloudflare's challenges cannot support the following:
```
Comment (ranbel) [inline content/firewall/cf-firewall-rules/cloudflare-challenges.md]:
```suggestion
Cloudflare challenges cannot support the following:
```

Comment (ranbel) [reply content/firewall/cf-firewall-rules/cloudflare-challenges.md]: should "challenges" be capitalized?

**Review by ranbel — APPROVED** [review]

File: content/firewall/cf-firewall-rules/cloudflare-challenges.md (hunk 2 above)
Comment (patriciasantaana) [reply content/firewall/cf-firewall-rules/cloudflare-challenges.md]: the title for the page isn't capitalized but thanks for calling out the apostrophe I missed!

**Pull request merged**